src/arm/bbb-bone-buses.dtsi

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2020 Deepak Khatri <deepaklorkhatri7@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include <dt-bindings/board/am335x-bbb-pins.h>
+
+&am33xx_pinmux {
+bone_uart_1_pins: pinmux_bone_uart_1_pins {
+		pinctrl-single,pins = <
+			P9_24( PIN_OUTPUT_PULLUP | MUX_MODE0)
```

Comment:
Having the consistent comments would be nice. pin.mode

Reply:
Agreed :+1: